### ISSUE TITLE
Privacy policy

### ISSUE THREAD

**Issue by stefnotch** [tuwien2020/tgi-pages#97]
(TGI Pages) Privacy Policy:
tl;dr: We don't store any info.

Technical information, for the curious:
- This site is open source on GitHub!
- This site is hosted on Github Pages, so you might want to look up their privacy policy. After all, your browser is connecting to their servers.
- It's mostly client-side, in other words, when you open the site, the site's code gets downloaded by your browser. Then, your browser happily runs and renders it, without needlessly needing to ask a server.

Disclaimer: This is not legal info and may be inaccurate. 
We do try to keep it up-to-date, and if you spot a mistake, pretty please tell us about it.
